lib/aurora_bootstrapper/utility.rb

```diff
@@ -0,0 +1,48 @@
+module AuroraBootstrapper
+  class Utility
```

Comment:
the ruby convention for blocks of class methods is
```ruby
module AuroraBootstrapper
  class Utility
    class << self
      def get_db_pass
        #...
      end
      
      #...
    end
  end
end
```

It helps visually separate class from instance methods